test/suites/clustering.sh

```diff
@@ -1217,6 +1217,14 @@ test_clustering_network() {
   LXD_DIR="${LXD_ONE_DIR}" lxc config device remove c3 eth0
   LXD_DIR="${LXD_ONE_DIR}" lxc config device add c3 eth0 nic hwaddr="${c1MAC}" nictype=bridged parent="${net}"
 
+  # Check networks local to a cluster member show up when targeting that cluster member
```

Comment:
the per-member dummy interfaces wont be being created inside the network namespace for each cluster member.

Reply:
See https://github.com/canonical/lxd/blob/a82bf163bf28f8c28e2e7e95e6201d0ce9bca0dd/test/includes/clustering.sh#L26

Reply:
Here's an example of running network setup inside a network namespace

https://github.com/canonical/lxd/blob/a82bf163bf28f8c28e2e7e95e6201d0ce9bca0dd/test/includes/clustering.sh#L77-L85

Reply:
and the pid file for the ns seems to be accessible here

https://github.com/canonical/lxd/blob/a82bf163bf28f8c28e2e7e95e6201d0ce9bca0dd/test/includes/clustering.sh#L70C9-L70C42

Reply:
Thanks for the pointers, with your help, I think the tests are fine now :)